runtime/doc/treesitter.txt

```diff
@@ -573,6 +573,14 @@ associated with patterns:
     • `injection.parent` - indicates that the captured node’s text should
       be parsed with the same language as the node's parent LanguageTree.
 
+Injection queries are currently run over the entire buffer, which can be slow
+for large buffers. To disable injections for a certain language, just place an
+empty query file in your 'runtimepath'. For example, to disable injections for
+`c`, use the following `queries/c/injections.scm`: >query
+
+    ; disabled
```

Comment:
This doesn't do anything right? Just the existence of the file disables the injections? Unlike `; inherits` and `; extends`.

If so then this is slightly confusing.

Reply:
Yeah, true. Should I just remove this example then? Maybe does more harm than good.

Reply:
I think it's better to explicitly describe what happens, and/or use a comment that us obviously arbitrary.